.swift-version

```diff
@@ -0,0 +1 @@
+5.5.1
```

Comment:
Given we are releasing with Xcode 12.X I don't think we can use 5.5.1 here.
But maybe now that Xcode 13.2 is out we should be able to use it for release as it should solve the compatibility issues we have found.

cc @fortmarek @luispadron

Reply:
Isn't this version used only for linting which is not run during release? I think if CI passes, we should be fine. However, it is worth a try whether we can start releasing with 13.2 as we could use Swift 5.5 features such as synthesization of Codable conformance for enums with associated values.

Reply:
If swiftformat relies on this for some checks, I imagine it means such API is available only on newer iOS versions 🤷‍♂️

Reply:
Not sure how iOS is related here? But it's true we could possibly run into a scenario where `swiftformat` autofixes code in a way that it becomes uncompilable with Xcode 12.X - but that should be caught by one of our checks.

Reply:
Yeah I meant swift version sorry :)
I think we should put here the minimum version we support Tuist on. If we bump the release to use Xcode 13.2, then we can bump this to 5.5.2 👍